src/main/webapp/profileScript.js

```diff
@@ -426,23 +426,27 @@ function displaySavedInterests(user, viewer) {
   savedInterestsContainer.innerHTML = '';
 
   const interestHeading = document.createElement('h1');
-  interestHeading.style = 'text-align: center';
   interestHeading.innerText = 'Saved Interests';
   savedInterestsContainer.appendChild(interestHeading);
 
+  const interestsList = document.createElement('div');
+  interestsList.id = 'interests-list';
+
   if (viewer === PROFILE_VIEWER_PERSONAL || viewer === PROFILE_VIEWER_BUDDY) {
     fetch('/interest').then(response => response.json()).then((interests) => {
       let interestCount = 0;
       for (let i = 0; i < interests.length; i ++) {
         if (interests[i].interestedUsers.includes(user.id)) {
-          savedInterestsContainer.appendChild(createInterest(interests[i]));
+          interestsList.appendChild(createInterest(interests[i]));
           interestCount ++;
         }
       }
       if (interestCount == 0) {
         const interestMessage = document.createElement('p');
         interestMessage.innerText = 'No interests to show.';
         savedInterestsContainer.appendChild(interestMessage);
+      } else {
+        savedInterestsContainer.appendChild(interestsList);
```

Comment:
Nit: You could obviate the need for `interestCount` with a guard clause at the top of the `fetch` callback:

```
if (interests.length === 0) {
  // append 'No interests to show.'
  return;
}

for (let i = 0; i < interests.length; i++) {
  // ...
}
```